fints/segments/depot.py

```diff
@@ -0,0 +1,19 @@
+from . import FinTS3Segment
+
+class HKWPD(FinTS3Segment):
+    """
+    HKWPD (Depotaufstellung anfordern)
+    Section C.4.3.1
+    Example: HKWPD:3:7+23456::280:10020030+USD+2'
+    """
+    type = 'HKWPD'
+
+    def __init__(self, segno, version, account):
+        self.version = version
+        data = [
+            account
+            #'EUR'        # Währung der Depotaufstellung"
+            #2             # Kursqualität
```

Comment:
Can you explain here a bit? The example from the docstring contains a currency and here you drop it, is it not required? Maybe leave a comment there why it's dropped for the future.